raccoongang/edx-platform
# Elastic APM support

**Description:** https://www.elastic.co/solutions/apm

It works for OSPP project.

**Configuration instructions:** Add to server-vars.yml:

```
EDXAPP_ENV_EXTRA:
  APM_SERVICE_NAME: 'My super project name for Kibana'
  APM_SERVER_URL: 'https://logs.raccoongang.com:8202'
EDXAPP_AUTH_EXTRA:
  APM_SECRET_TOKEN: 'super secret token from 1password'
```

**Testing instructions:**

1. Install APM client to edx-platform (current PE) using edx update procedure
2. Add configuration to server-vars.yml
3. Wait 5-10 minutes to collect data.
4. Check the APM tab in the Kibana to find your host by name from APM_SERVICE_NAME.

**Merge checklist:**

**Post merge:**
